data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -1,7 +1,9 @@
 import decimal
+import src.blockchain_call
```

Comment:
Move to local imports.

Reply:
Fixed [0f9d33c](https://github.com/CarmineOptions/derisk-research/pull/121/commits/0f9d33cec77dcdb69c82d4f7947cde07649e2089)
pls check and let me know 😄

Reply:
`src` is part of this project, move it one block below to where other local import are 🙏🏼

Reply:
Fix it in this commit: 73f712a53242c2402dbad84e335c8b84ae1e3ebf please let me know